birdhouse/components/README.rst

```diff
@@ -8,6 +8,76 @@ Birdhouse Components
 Scheduler
 =========
 
+The scheduler component runs specific jobs on a schedule. This is similar to using the `cron`
+service but this runs in docker containers and is specifically designed to interact with the
+Birdhouse stack.
+
+Scheduler jobs can be enabled by sourcing specific ``*.env`` files in the ``components/scheduler``
+directory. The jobs that come included with the Birdhouse code are as follows:
+
+* Enable automatic deployment:
+  * additional description of this can be found in the :ref:`Automated Deployment` section.
+  * to enable this component add the following to your local environment file:
+    .. code-block:: shell
+      . $COMPOSE_DIR/components/scheduler/autodeploy_job.env
+
+* Automatically rotate logs:
+  * to enable this component add the following to your local environment file:
+    .. code-block:: shell
+      . $COMPOSE_DIR/components/scheduler/logrotate_job.env
+
+* Automatically update tutorial Jupyter notebooks:
+  * note that this only can be enabled if the ``jupyterhub`` component is also enabled
```

Comment:
Should the dependency be applied dynamically when sourcing its env file?

This could be relevant if the sourced env makes use of a `jupyterhub`-specific config that must be sourced first.

Same concept of the below `thredds` cases.